front-end/src/app/player/player.component.css

```diff
@@ -0,0 +1,21 @@
+.player-card {
+    background-color: #e4f7e9;
```

Comment:
Is there a close, named, colour we could use instead?

Reply:
I will make a colours file to create a name